packages/x-tailwindcss/src/x-tailwind-plugin/components/badge/light.ts

```diff
@@ -0,0 +1,16 @@
+import { TailwindHelpers } from '../../../types';
+
+/**
+ * Returns the `light` variant for component `badge`.
+ *
+ * @param helpers - The {@link TailwindHelpers} to generate CSS.
+ * @returns The {@link CssStyleOptions} for the variant.
+ */
+export function badgeLight({ theme }: TailwindHelpers) {
+  return {
+    light: {
+      backgroundColor: 'var(--badge-color-25)',
+      color: theme('colors.neutral.90')
```

Comment:
This doesn't match with the design, but I think the design is not right, isn't it?

![image](https://user-images.githubusercontent.com/77450928/224265576-7f5a2c3f-dd11-46da-919a-56195fd99dd6.png)

Reply:
Yep, I will leave a comment there to update the designs.